internal/apis/certmanager/types_certificate.go

```diff
@@ -252,15 +254,15 @@ type CertificateSpec struct {
 	NameConstraints *NameConstraints
 }
 
-type OtherNameSAN struct {
+type OtherName struct {
 	// OID is the object identifier for the otherName SAN.
 	// The object identifier must be expressed as a dotted string, for
-	// example, "1.2.840.113549.1.9.1".
-	OID string
+	// example, "1.2.840.113556.1.4.221".
+	OID string `json:"oid,omitempty"`
 
 	// Utf8Value is the string value of the otherName SAN.
 	// The string value represents a UTF-8 encoded asn1 value.
-	Utf8Value string
+	UTF8Value string `json:"UTF8Value,omitempty"`
```

Comment:
```suggestion
	UTF8Value string `json:"utf8Value,omitempty"`
```

Reply:
even though it's a small one, i disagree, since it's an acronym it should be UTF8, no? for example the WIKI page and standard all say  has https://en.wikipedia.org/wiki/UTF-8 https://www.utf8.com/ (although the URL is not capped)

Anyways i think they are both equally valid, why do you think the YAML shouldn't use that case?

Reply:
I take that back because we didn't capitalise OID above hehe